katz-rx/src/main/kotlin/katz/rx/observable/instances/ObservableMonad.kt

```diff
@@ -0,0 +1,18 @@
+package katz
+
+import io.reactivex.*
+
+interface ObservableMonad : Monad<ObservableWrap.F> {
+    override fun <A> pure(a: A): ObservableWrap<A> = ObservableWrap(Observable.just(a))
```

Comment:
There is an implicit requirement for monads: they have to override the map or the ap function, as they call into each other recursively because each can be expressed in terms of each other. Given that we already have map in the data type, that'd be my suggestion :D